modules/git/hook.go

```diff
@@ -1,5 +1,4 @@
-// Copyright 2015 The Gogs Authors. All rights reserved.
-// Copyright 2021 The Gitea Authors. All rights reserved.
+// Copyright 2024 The Gitea Authors. All rights reserved.
```

Comment:
```suggestion
// Copyright 2015 The Gogs Authors. All rights reserved.
// Copyright 2021 The Gitea Authors. All rights reserved.
```
We've decided not to change our existing copyright headers once they have been added.
Plus, removing the Gogs one is probably legally impossible.